### DISCUSSION TITLE
HTML help

### DISCUSSION THREAD

**Discussion by yolobolo69** [community/community#142553] [Programming Help]
### Body

how i make my html page auto reload after editing code

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mostaphabr** [comment]
use liveserver extention in your vs code

**Comment by yolobolo69** [comment]
it worked thanks man

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142555#discussioncomment-11055494) for reference to why this discussion was closed and locked. Thanks!
